modules/flatgeobuf/src/lib/parse-flatgeobuf.js

```diff
@@ -1,5 +1,27 @@
-// import {deserialize} from 'flatgeobuf/lib/geojson/featurecollection';
-import {deserialize, deserializeStream} from 'flatgeobuf/dist/flatgeobuf-geojson.min';
+import {
+  deserialize as deserializeGeoJson,
+  deserializeStream as deserializeStreamGeoJson
+} from 'flatgeobuf/dist/flatgeobuf-geojson.min';
+import {
+  deserialize as deserializeGeneric,
+  deserializeStream as deserializeStreamGeneric
+} from 'flatgeobuf/dist/flatgeobuf.min';
```

Comment:
Re https://github.com/visgl/loaders.gl/pull/884/files#r466768308, 

> > to support geojson and binary parsing we need to take from two overlapping dist bundles.

> Why is that ?

There are three `dist` bundles exported from `flatgeobuf`: 

- `flatgeobuf/dist/flatgeobuf-ol.min.js`: parses Flatgeobuf to OpenLayers objects
- `flatgeobuf/dist/flatgeobuf-geojson.min.js`: parses Flatgeobuf to GeoJSON objects
- `flatgeobuf/dist/flatgeobuf.min.js`: the _generic_ distribution. You pass in a function that manually parses the `flatbuffers` objects

If you want simple two-line parsing to geojson, you use the `flatgeobuf-geojson.min.js` file. If you want to parse directly to binary, you need the `flatgeobuf.min.js` file where you can pass your own parsing function.

The ways around this are either: 

- only use `flatgeobuf.min.js` and copy all the geojson parsing code from the flatgeobuf repo
- Import from the uncompiled ES6 `flatgeobuf/lib/*` and let our build tool do the tree shaking

The latter would be preferable, but I was having issues loading from `flatgeobuf/lib/*` in Node.